docs/tutorials/dev_add_test_group.md

```diff
@@ -2061,26 +2060,19 @@ class Forward(OceanModelStep):
         cell_count : int or None
             The approximate number of cells in the mesh
         """
-        if at_setup:
-            # no file to read from, so we'll compute it based on config options
-            section = self.config['yet_another_channel']
-            lx = section.getfloat('lx')
-            ly = section.getfloat('ly')
-            nx, ny = compute_planar_hex_nx_ny(lx, ly, self.resolution)
-            cell_count = nx * ny
-        else:
-            # get nCells from the input file
-            with xr.open_dataset('initial_state.nc') as ds:
-                cell_count = ds.sizes['nCells']
+        section = self.config['yet_another_channel']
+        lx = section.getfloat('lx')
+        ly = section.getfloat('ly')
+        nx, ny = compute_planar_hex_nx_ny(lx, ly, self.resolution)
+        cell_count = nx * ny
         return cell_count
 ```
-This method gets called once at setup (with `at_setup=True`) and once at
-runtime (with `at_setup=False`).  We do this because we need to compute or 
-estimate the size of the mesh during setup so we have a good guess at the
-resources it will need when we add it to a test suite and make a job script for
-running it.  Then, at runtime, we may need to revise that size, either because
-it was an estimate only or because it was based on config options that a user
-may have changed after setup.  Here, we use 
+This method gets called once at setup and once at runtime.  We do this because 
+we need to compute or estimate the size of the mesh during setup so we have a 
+good guess at the resources it will need when we add it to a test suite and 
+make a job script for running it.  Then, at runtime, we may need to revise that
+size because it was based on config options that a user may have changed after
+setup.  Here, we use 
 {py:func}`polaris.mesh.planar.compute_planar_hex_nx_ny()` to get `nx` and `ny`
 (and thus the total cell count) during setup because we have no other way to
 get them (the initial condition has not yet been created, for example).  But
```

Comment:
```suggestion
(and thus the total cell count) during setup and at runtime when we use 
task parallelism because we have no other way to
get them (the initial condition has not yet been created, for example).
```